documentation/CLI-for-JFrog-Curation.md

```diff
@@ -0,0 +1,81 @@
+JFrog CLI : CLI for JFrog Curation
+======================================
+
+
+Overview
+--------
+
+This page describes how to use JFrog CLI with Jfrog Curation.
+
+Read more about JFrog CLI [here](https://jfrog.com/help/r/jfrog-cli).
+
+---
+**Note**
+> JFrog Curation is only available since [Artifactory 7.63.2](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.63.2-cloud) And [Xray 3.78.9](https://jfrog.com/help/r/jfrog-release-information/xray-3.78.9).
+
+---
+
+### Syntax
+
+When used with JFrog Distribution, JFrog CLI uses the following syntax:
+
+	$ jf ca command-name command-options 
+
+Where:
+
+
+|                 |                                                                                                 |
+|-----------------|-------------------------------------------------------------------------------------------------|
+| command-name    | The command to execute. Note that you can use either the full command name or its abbreviation. |
+| command-options | A set of options corresponding to the command                                                   |
+
+
+
+### Commands
+
+The following sections describe the commands available in the JFrog CLI for use with JFrog Curation.
+
+Curation-Audit
+---------------------
+**Note**
+>The command _curation-audit_ supports only [npm](https://www.npmjs.com/) projects.
+
+The _jf curation-audit_ command enables developers to scan project dependencies to find packages that were blocked by the JFrog curation service. This command provides developers with more detailed information, such as whether the blocked package is the project’s direct dependency or is a transitive dependency. This information helps developers to resolve blocked packages more efficiently as they will be able to make a more informative decision based on what Policy violation occurred and what exactly needs to be resolved.
+
+For each blocked package the CLI provides the violated curation Policies, The command builds a deep dependencies graph for the project, and requests the curation status by a HEAD request for each node in the tree. It uses the package manager that is used in the project to build the dependencies graph.
```

Comment:
Let's use Curation instead of curation when possible
```suggestion
For each blocked package the CLI provides the violated Curation Policies, The command builds a deep dependencies graph for the project, and requests the Curation status by a HEAD request for each node in the tree. It uses the package manager that is used in the project to build the dependencies graph.
```